crypto/x86_64cpuid.pl

```diff
@@ -145,8 +145,19 @@
 	or	\$0x40000000,%edx	# set reserved bit#30 on Intel CPUs
 	and	\$15,%ah
 	cmp	\$15,%ah		# examine Family ID
-	jne	.Lnotintel
+	jne	.LnotP4
 	or	\$0x00100000,%edx	# set reserved bit#20 to engage RC4_CHAR
+.LnotP4:
+	cmp	\$6,%ah
+	jne	.Lnotintel
+	and	\$0x0ffff0f0,%eax
```

Comment:
> Did you mean 0x0fff0ff0?

Yes. Thanks! Request is coming up...

> (Checking 6 is also redundant with lines 151 and 152.)

Yeah, but there is no side effect if one double-checks. Idea was to make it look close to corresponding entry in cpuid table in sde.